addons/skin.confluence/720p/DialogConfirm.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<window>
+	<coordinates>
+		<left>315</left>
+		<top>230</top>
+	</coordinates>
+	<include>dialogeffect</include>
+	<depth>DepthDialog+</depth>
```

Comment:
yup, depths are defined like that in confluence:
https://github.com/xbmc/xbmc/blob/master/addons/skin.confluence/720p/includes.xml#L21